src/sphinx/formats/universal.rst

```diff
@@ -158,6 +158,12 @@ If you want to force local for the `tgz` output add this line:
   universalArchiveOptions in (Universal, packageZipTarball) := Seq("--force-local", "-pcvf")
 
 This will set the cli options for the `packageZipTarball` task in the `Universal` plugin to use the options ``--force-local`` and ``pcvf``.
+Be aware that the above line will overwrite the default options.  You may want to prepend your options, doing something like:
+
+.. code-block:: scala
+
+  universalArchiveOptions in (Universal, packageZipTarball) :=
```

Comment:
I tried the append as

    universalArchiveOptions in (Universal, packageZipTarball) ++= Seq("--exclude", "*~")

and the resulting output included:

    Running with tar -pcvf --exclude *~ /tmp/sbt_72102b69/myApp-0.1.tar myApp-0.1
    tar: *~: Cannot stat: No such file or directory
    tar: /tmp/sbt_72102b69/myApp-0.1.tar: Cannot stat: No such file or directory

and then later:

    gzip: can't stat: /tmp/sbt_72102b69/myApp-0.1.tar: No such file or directory
    java.lang.RuntimeException: Error gziping /tmp/sbt_72102b69/myApp-0.1.tar. Exit code: 1
    [error] (server/universal:packageZipTarball) Error gziping /tmp/sbt_72102b69/myApp-0.1.tar. Exit code: 1

Correct me if I'm wrong, but it looks like the generated command told `tar` to name the tar-file `--exclude`, and that files named `*~` and `/tmp/sbt_72102b69/myApp-0.1.tar` are to be included in the archive.

Again, the default options seem to depend on nothing coming between the `-f` switch and the tar-file name, which is what appending--but not prepending--custom options to the defaults does.